### ISSUE TITLE
Release pretrained model for DepthSplat on Re10K

### ISSUE THREAD

**Issue by WalterC9** [ziplab/ZPressor#8]
Hi, 

Thank you for such a great work and release the code. 

I tried to finetune the pretrained model depthsplat-dl3dv-zpressor-n50-256x448 on DL3DV to understand the training, but it always results in nan loss. Would it be possible to also release the pretrained DepthSplat model on Re10K. 

Thank you for help on this!

**Comment by lhmd** [comment]
Hi @WalterC9 ! I'm sorry, I've lost these files; they should be on a server that's currently unreachable. I'd like to know which loss value is NaN, because it might just be due to usage issues and not necessarily a problem with the pre-trained weights.
